packages/components/src/color-palette/index.native.js

```diff
@@ -40,7 +40,12 @@ function ColorPalette( {
 	currentSegment,
 	onCustomPress,
 	shouldEnableBottomSheetScroll,
+	shouldShowCustomIndicatorOption = true,
+	shouldShowCustomLabel = true,
+	customStyles,
 } ) {
+	const styles = customStyles === undefined ? defaultStyles : customStyles;
```

Comment:
Related to https://github.com/WordPress/gutenberg/pull/23833/files#r454505493 what do we want to customize here?